README.md

```diff
@@ -1 +1,98 @@
-# Homes-Rental
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# House Rental
```

Comment:
- [ ] Your project is beautiful.  Please, kindly show it off by adding a screenshot on your readme. 💪